contracts/interfaces/IConstAddressDeployer.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title IConstAddressDeployer Interface
+ * @notice This interface defines the contract responsible for deploying and optionally initializing new contracts
+ * using the `CREATE2` method.
+ */
+// slither-disable-next-line missing-inheritance
```

Comment:
this can probably be removed after inheriting in Create2Deployer

Reply:
removed